tests/cases/fourslash/incompleteFunctionCallCodefixTypeParameterVariable.ts

```diff
@@ -19,24 +19,24 @@ verify.codeFix({
   added2(et)
 }
 
-function added1(et: string) {
+function added1<T extends "phone" | "home">(et: T) {
     throw new Error("Function not implemented.")
 }
 `,
 });
 
 goTo.marker("2");
 verify.codeFix({
-  description: "Add missing function declaration 'added1'",
-  index: 0,
+  description: "Add missing function declaration 'added2'",
+  index: 1,
```

Comment:
this was a mistake in the original test case, `goTo.marker("2")` has no effect on `verify.codeFix`, we need to use a global index here to test the codefix at that marker